### PR TITLE
decouple tcli from natlab

### PR DESCRIPTION
**Pull request by gytsto**
### Problem
tcli is not needed anymore in natlab since pyro has been introduced. and tcli is actually messing up things with logs and other stuff.

### Solution
remove tcli from natlab

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] Functionality is covered by unit or integration tests

### CODE REVIEW

File: src/device.rs [hunk 1]
```diff
@@ -799,6 +803,15 @@ impl Device {
         })
     }
 
+    pub fn receive_ping(&self) -> Result<String> {
```
Comment (matislovas) [inline src/device.rs:806]: Wild idea: maybe we should expose/compile API methods, that are only used in tests, only in tests? I know that there was a libtelio space saving initiative :smile:

Comment (mathiaspeters) [reply src/device.rs:806]: Uniffi doesn't like conditional compilation like that unfortunately

**Comment by mathiaspeters** [comment]
Nit: the name of the changelog file should just be the ticket number. Since the file is empty it doesn't matter much, just to keep to convention

File: nat-lab/pyproject.toml [hunk 2]
```diff
@@ -3,7 +3,7 @@ name = "nat-lab"
 
 [tool.black]
 preview = true
-exclude="tests/protobuf"
+exclude = "tests/protobuf,tests/uniffi"
```
Comment (mathiaspeters) [inline nat-lab/pyproject.toml:6]: Will this not make all the python tools skip `libtelio_remote.py` and `libtelio_proxy.py`?

File: src/device.rs (hunk 1 above)
Comment (gytsto) [reply src/device.rs:806]: Not that wild, but good idea to not expose those methods in release. But will create separate issue for this if you don't mind.

File: nat-lab/pyproject.toml (hunk 2 above)
Comment (gytsto) [reply nat-lab/pyproject.toml:6]: you're right, will change so it would only skip generated part

File: src/device.rs (hunk 1 above)
Comment (gytsto) [reply src/device.rs:806]: LLT-5451

File: nat-lab/pyproject.toml (hunk 2 above)
Comment (gytsto) [reply nat-lab/pyproject.toml]: fixed

**Review by mathiaspeters — COMMENTED** [review]
+1

**Pull request merged**